src/Analyzers/MSTest.Analyzers.CodeFixes/CodeFixResources.resx

```diff
@@ -123,4 +123,7 @@
   <data name="AssemblyInitializeShouldBeValidCodeFix" xml:space="preserve">
     <value>Fix signature</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="FixAssertionArgsOrder" xml:space="preserve">
+    <value>Fix assertion args order</value>
```

Comment:
I would go with either `Swap actual/expected arguments` or `Fix actual/expected arguments order`.

cc @nohwnd

Reply:
I used `Fix actual/expected arguments order` for now